app/models/game.rb

```diff
@@ -1,4 +1,4 @@
 class Game < ApplicationRecord
   has_many	:pieces
-  has_many	:users
+
```

Comment:
Looks solid, what is missing here on the game model is a reference to the balck and white players...

sth like `belongs_to :white_player, :class_name => 'User', :foreign_key => 'white_player_id'`. etc...